testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/EEConcurrencyExecutorShutdownTestCase.java

```diff
@@ -0,0 +1,208 @@
+        ModelNode addContextService = createOpNode("profile=default/subsystem=ee/context-service=default", ADD);
+        addContextService.get("jndi-name").set("java:jboss/ee/concurrency/context/default");
+        addContextService.get("use-transaction-setup-provider").set(true);
+        executeForResult(addContextService, masterClient);
+        // /profile=default/subsystem=ee/managed-executor-service=default:add(jndi-name=java:jboss/ee/concurrency/executor/default, context-service=default, hung-task-threshold=60000, keepalive-time=5000)
+        ModelNode addManagedExecutorService = createOpNode("profile=default/subsystem=ee/managed-executor-service=default", ADD);
+        addManagedExecutorService.get("jndi-name").set("java:jboss/ee/concurrency/executor/default");
+        addManagedExecutorService.get("context-service").set("default");
+        addManagedExecutorService.get("hung-task-threshold").set(60000);
+        addManagedExecutorService.get("keepalive-time").set(5000);
+        executeForResult(addManagedExecutorService, masterClient);
+    }
+
+    /**
+     * Tests if the server with running ConcurrencyExecutor can be stopped using cli command
+     * /host=master/server-config=server-one:stop(timeout=0)
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testConcurrencyExecutorShutdown() throws Exception {
+        ModelNode content = new ModelNode();
+        content.get("archive").set(true);
+        content.get("path").set(new File(tmpDir, "archives/" + ARCHIVE_FILE_NAME).getAbsolutePath());
+        ModelNode deploymentOpMain = createDeploymentOperation(content, MAIN_SERVER_GROUP_DEPLOYMENT_ADDRESS);
+        executeForResult(deploymentOpMain, masterClient);
+
+        this.stopServer(FIRST_SERVER_NAME);
+    }
+
+    private ModelNode createDeploymentOperation(ModelNode content, PathAddress... serverGroupAddressses) {
+        ModelNode composite = createEmptyOperation(COMPOSITE, PathAddress.EMPTY_ADDRESS);
+        ModelNode steps = composite.get(STEPS);
+        ModelNode step1 = steps.add();
+        step1.set(createAddOperation(ROOT_DEPLOYMENT_ADDRESS));
+        step1.get(CONTENT).add(content);
+        for (PathAddress serverGroup : serverGroupAddressses) {
+            ModelNode sg = steps.add();
+            sg.set(createAddOperation(serverGroup));
+            sg.get(ENABLED).set(true);
+        }
+        return composite;
+    }
+
+    /**
+     * Stops the given server and waits until STOPPED status
+     *
+     * @param serverName
+     * @throws Exception
+     */
+    private void stopServer(final String serverName) {
+        ModelNode op = new ModelNode();
+        op.get(OP_ADDR).add(HOST, MASTER);
+        op.get(OP_ADDR).add(SERVER_CONFIG, FIRST_SERVER_NAME);
+        op.get(OP).set(STOP);
+        op.get(TIMEOUT).set(0);
+        op.get(ADMIN_ONLY).set(false);
```

Comment:
Although the operation is executed, this is not a valid param, so you should not use it, see ` /host=master/server-config=server-one:read-operation-description(name=stop)`